dataland-frontend/tests/e2e/utils/Auth.ts

```diff
@@ -14,7 +14,7 @@ export function logout(): void {
     .wait(1000)
```

Comment:
To make sure that there was sufficient time to react. If the url was `/` before and should change here this assertion would go through. I think this is supposed to be prevented by this.

Reply:
Ok but the first "wait" simply waits for the dropdown to open, which does not make sense, because the "cy.get" on the logout anchor will wait right until it is there.

And the second "wait" does not make sense because obviously cypress is on the "/companies" page when it logs out, so the original url has "/companies" at the end. So the assertion won't go through.

Reply:
We will remove the wait-statements once the CI is green to check if it breaks smth.

Reply:
I optimized the logout method and the tests are green.